reana-ui/src/components/WorkflowActionsPopup.js

```diff
@@ -69,6 +70,17 @@ export default function WorkflowActionsPopup({ workflow, className }) {
     });
   }
 
+  menuItems.push({
+    key: "share",
+    content: "Share workflow",
```

Comment:
FWIW when testing the PR set:

-   r-client: 692
-   r-client-go: 153
-   r-ui: 375
-   r-server: 658
-   r-workflow-controller: 552
-   r-commons: 429
-   r-db: 216

The workflow sharing modal was creating a  trackeback on the r-server side:

```
2024-03-28 13:52:43,318 | root | uWSGIWorker2Core1 | ERROR | share_workflow does not have parameter user
Traceback (most recent call last):
  File "/code/./reana_server/rest/workflows.py", line 3397, in share_workflow
    response, http_response = current_rwc_api_client.api.share_workflow(
  File "/usr/local/lib/python3.8/dist-packages/bravado/client.py", line 246, in __call__
    request_params = construct_request(
  File "/usr/local/lib/python3.8/dist-packages/bravado/client.py", line 285, in construct_request
    construct_params(operation, request, op_kwargs)
  File "/usr/local/lib/python3.8/dist-packages/bravado/client.py", line 305, in construct_params
    raise SwaggerMappingError(
bravado_core.exception.SwaggerMappingError: share_workflow does not have parameter user
```

This is because the r-ui PR was not updated yet to send the sharing details via the JSON body rather than via the URL query (used previously).